internal/integtest/helpers.go

```diff
@@ -313,6 +313,19 @@ func MakeObjectWithAnnotation(ctx context.Context, resource string, nsName,
 	createdObjects = append(createdObjects, inst)
 }
 
+// MakeObjectWithLabel creates an empty object with label given kind in a specific
+// namespace. The kind and its corresponding GVK should be included in the GVKs map.
+func MakeObjectWithLabel(ctx context.Context, resource string, nsName,
```

Comment:
Should be plural (e.g. `MakeObjectWithLabels`) since it can accept multiple labels. Feel free to fix the function above as well if you feel like it but you don't have to :)